cns/service/main.go

```diff
@@ -197,7 +197,8 @@ func main() {
 		return
 	}
 
-	if logger := log.GetStd(); logger != nil {
+	// Set-up channel for CNS telemetry if it's enabled (enabled by default)
+	if logger := log.GetStd(); logger != nil && acn.GetArg(acn.OptTelemetry).(bool) {
```

Comment:
acn.GetArg(acn.OptTelemetry).(bool). This is getting called at 2 places..call this at top as like other args and use that variable

Reply:
done